src/common/api_impl.cc

```diff
@@ -1095,12 +1098,7 @@ void BindMonomorphicApi(PJRT_Api *api) {
     return nullptr;
   };
 
-  api->PJRT_Plugin_Attributes =
-      +[](PJRT_Plugin_Attributes_Args *args) -> PJRT_Error * {
-    DLOG_F(LOG_DEBUG, "PJRT_Plugin_Attributes");
-    args->num_attributes = 0;
-    return nullptr;
-  };
+  api->PJRT_Plugin_Attributes = InitializePluginAttributes;
```

Comment:
Should we move this to the `PluginAttributes::BindApi(api)` function, to be in line with the other bindings? We could do the same for `PJRT_Plugin_Initialize`

Reply:
That's a good question, I wasn't sure if we want to couple API bindings with these classes as it was currently done so I've deferred that decision for later. Could you please add this to the list you are compiling, of things to consider for API implementation changes (to rethink where to do API bindings)?

Reply:
Okay, I will do that!